src/base/rss/rss_article.cpp

```diff
@@ -112,6 +112,11 @@ void Article::markAsRead()
     }
 }
 
+void Article::markAsUnread()
```

Comment:
I wrote it to make testing easier, but it is the only way to undo an accidental Mark All Read.